packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-validation.spec.ts

```diff
@@ -0,0 +1,350 @@
+/**
+ * @file Error Handling Tests for CloudPulse Dashboard.
+ */
+import { mockAppendFeatureFlags } from 'support/intercepts/feature-flags';
+import {
+  mockCreateCloudPulseJWEToken,
+  mockGetCloudPulseDashboard,
+  mockGetCloudPulseDashboards,
+  mockGetCloudPulseMetricDefinitions,
+  mockGetCloudPulseServices,
+} from 'support/intercepts/cloudpulse';
+import { ui } from 'support/ui';
+import { widgetDetails } from 'support/constants/widgets';
+import {
+  accountFactory,
+  dashboardFactory,
+  dashboardMetricFactory,
+  databaseFactory,
+  regionFactory,
+  widgetFactory,
+} from 'src/factories';
+import { mockGetUserPreferences } from 'support/intercepts/profile';
+import { mockGetRegions } from 'support/intercepts/regions';
+import { extendRegion } from 'support/util/regions';
+import { mockGetDatabases } from 'support/intercepts/databases';
+import { apiMatcher } from 'support/util/intercepts';
+import { Database } from '@linode/api-v4';
+import { mockGetAccount } from 'support/intercepts/account';
+
+const {
+  metrics,
+  id,
+  serviceType,
+  dashboardName,
+  region,
+  engine,
+  clusterName,
+  nodeType,
+} = widgetDetails.dbaas;
+
+const dashboard = dashboardFactory.build({
+  label: dashboardName,
+  service_type: serviceType,
+  widgets: metrics.map(({ title, yLabel, name, unit }) => {
+    return widgetFactory.build({
+      label: title,
+      y_label: yLabel,
+      metric: name,
+      unit,
+    });
+  }),
+});
+
+const metricDefinitions = {
+  data: metrics.map(({ title, name, unit }) =>
+    dashboardMetricFactory.build({
+      label: title,
+      metric: name,
+      unit,
+    })
+  ),
+};
+
+const mockRegion = extendRegion(
+  regionFactory.build({
+    capabilities: ['Linodes'],
+    id: 'us-ord',
+    label: 'Chicago, IL',
+    country: 'us',
+  })
+);
```

Comment:
```suggestion
const mockRegion = regionFactory.build({
  capabilities: ['Linodes'],
  id: 'us-ord',
  label: 'Chicago, IL',
  country: 'us',
});
```
We don't need `extendRegion` here. The `ExtendedRegion` type is used internally by our utils when we need to compare a region's API label against what gets displayed in Cloud Manager (e.g. `Chicago, IL` vs `US, IL, Chicago`). I can go into more detail about this and how it works, but generally speaking you can completely ignore the `ExtendedRegion` type when writing your tests.